evalai/utils/challenges.py

```diff
@@ -667,3 +669,54 @@ def display_leaderboard(challenge_id, phase_split_id):
         pretty_print_leaderboard_data(attributes, results)
     else:
         echo(style("Sorry, no Leaderboard results found.", bold=True, fg="red"))
+
+
+def upload_annotations_file_with_presigned_url(challenge_phase_pk, file):
+    url = "{}{}".format(get_host_url(), URLS.get_annotation_file_presigned_url.value)
+    url = url.format(challenge_phase_pk)
+    headers = get_request_header()
+
+    try:
+        # Fetching the presigned url.
+        data = {"file_name": file}
+        response = requests.get(url, headers=headers, data=data)
+        if response.status_code is not HTTPStatus.OK:
+            response.raise_for_status()
+
+        response = response.json()
+        presigned_url = response.get("presigned_url")
+
+        # Uploading the annotation file for the current phase to S3.
+        response = upload_with_presigned_url(file, presigned_url)
+    except requests.exceptions.HTTPError as err:
+        if response.status_code in EVALAI_ERROR_CODES:
+            validate_token(response.json())
+            echo(
+                style(
+                    "Error: {}".format(response.json()["error"]),
```

Comment:
shouldn't this message be more descriptive? similar to what was used for submission error message